website/content/docs/k8s/operations/faq.mdx

```diff
@@ -0,0 +1,21 @@
+---
+layout: docs
+page_title: FAQ
+sidebar_title: FAQ
+description: Frequently asked questions
+---
+
+# FAQ
```

Comment:
Do you think this should be a section under Consul's main [FAQ](https://www.consul.io/docs/troubleshoot/faq) page? I think this would make it more discoverable. Or is your thinking to have an FAQ page specific to operations?